### ISSUE TITLE
Livechat in rocket.chat 2.1.1 does not load in IE11

### ISSUE THREAD

**Issue by glangePDM** [RocketChat/Rocket.Chat#15819]
### Description:

We have recently installed the latest version 2.1.1 of rocket.chat with snap. Everything is fine but the livechat widget is not loading in frontend in IE11 on Windows 7 or Windows 10. In Edge 15 or higher the widget is loading.

We have got an syntax error in console output.

Our rocket.chat server is running on Ubuntu 18.4

### Steps to reproduce:

1. Activate livechat in admin panel
2. Go to frontend
3. IE11 shows a "Syntax Error" in console output but no widget

### Expected behavior:

Widget within the iframe is loading like in Edge or other browsers.

### Server Setup Information:

- Version of Rocket.Chat Server: 2.1.1
- Operating System: Ubuntu 18.04
- Deployment Method: Snap
- Number of Running Instances: 1
- NodeJS Version: v8.15.1
- MongoDB Version: 3.6.14

### Client Setup Information

- Desktop App or Browser Version: IE11
- Operating System: Windows 7 and 10

### Error message in IE11 console

Script1002: Syntax error
rocketchat-livechat.min.js (7,3021)

**Comment by renatobecker** [comment]
@glangePDM What is exactly the error you're getting on your browser console?

**Comment by glangePDM** [comment]
I only get the message like described above.

"Script1002: Syntax error
rocketchat-livechat.min.js (7,3021)"

I think it's easy to reproduce. We testet it with IE11 in Windows 7 and 10.

**Comment by renatobecker** [comment]
> I only get the message like described above.
> 
> "Script1002: Syntax error
> rocketchat-livechat.min.js (7,3021)"
> 
> I think it's easy to reproduce. We testet it with IE11 in Windows 7 and 10.

Yeah but you can expand the error and then you'll see the part of the code which is raising this error.

**Comment by glangePDM** [comment]
> > I only get the message like described above.
> > "Script1002: Syntax error
> > rocketchat-livechat.min.js (7,3021)"
> > I think it's easy to reproduce. We testet it with IE11 in Windows 7 and 10.
> 
> Yeah but you can expand the error and then you'll see the part of the code which is raising this error.

If I click on the error message I get a marker in line 7. See the screenshot attached. I hope this helps to identify the problem.
<img width="1660" alt="screen_rocket chat_2 1 1_livechat_IE11" src="https://user-images.githubusercontent.com/49510112/69485600-deb09900-0e41-11ea-9fbc-3da627e98cdd.png">

**Comment by pinalpithadiya** [comment]
Did anyone get the solution for this?
 I am getting the same problem with IE11.
 I think the issue is with s=()=>{}  which will not get resolved by IE11 can anyone fix this and update the repo?

Thanks in advance.
